static/lang/en.json

```diff
@@ -290,6 +290,20 @@
                 "Simple": "Simple",
                 "TitleEdit": "{name} (Edit Mode)"
             },
+            "Inventory": {
+                "ItemQuantity": {
+                    "Decrease": {
+                        "Click": "[Click] decreases by 1",
+                        "ShiftClick": "[Shift-Click] decreases by 5",
+                        "ControlClick": "[Control-Click] decreases by 10"
+                    },
+                    "Increase": {
+                        "Click": "[Click] increases by 1",
+                        "ShiftClick": "[Shift-Click] increases by 5",
+                        "ControlClick": "[Control-Click] increases by 10"
```

Comment:
```suggestion
                        "Click": "[Click] Decrease by 1",
                        "ShiftClick": "[Shift-Click] Decrease by 5",
                        "ControlClick": "[Control-Click] Decrease by 10"
                    },
                    "Increase": {
                        "Click": "[Click] Increase by 1",
                        "ShiftClick": "[Shift-Click] Increase by 5",
                        "ControlClick": "[Control-Click] Increase by 10"
```

Reply:
Why you always toggling this off :sob: 
![image](https://github.com/foundryvtt/pf2e/assets/106829671/9de3d7d1-8802-4526-a605-3500f9dd0b1a)